### PR TITLE
Contributing file

### LINKED ISSUE #783: New CONTRIBUTING file
We need a CONTRIBUTING(.md) file to be more opened to contributions!

### PR DESCRIPTION
**Pull request by marienfressinaud**
Create a CONTRIBUTING file. It gives info about:
- mailing lists
- report / fix bugs
- submit ideas
- contribution to i18n / documentation

Fix #783 

Ready for reviews:!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Translations are present in the subdirectories of `./app/i18n/`.
+
+We are working on a better way to handle internationalization but don't hesitate to suggest any idea!
+
+## Contribute to documentation
+
+The documentation needs a lot of improvements in order to be more useful to new contributors and we are working on it. If you want to give some help, meet us on [the dedicated repository](https://github.com/FreshRSS/documentation)!
+
+The best is to open tickets to ask your questions or to do any suggestions.
```
Comment (aledeg) [inline CONTRIBUTING.md]: I find "The best is to open tickets to ask your questions or to do any suggestions." is really French. I would just remove it.

Comment (marienfressinaud) [inline CONTRIBUTING.md]: Sure, it's not necessary.

Comment (marienfressinaud) [inline CONTRIBUTING.md]: Fixed

**Comment by marienfressinaud** [comment]
I was considering to merge this file in the beta branch too since it is the default branch and it should add [a message when creating a new ticket](https://github.com/blog/1184-contributing-guidelines) to inform about contributing guidelines.

**Comment by aledeg** [comment]
Sure, make an other PR

**Comment by marienfressinaud** [comment]
I'll use `git cherry-pick` instead since the contributing branch has been rebased on dev one.

Edit: done!

**Pull request merged**